sympy/stats/frv_types.py

```diff
@@ -314,6 +317,64 @@ def Binomial(name, n, p, succ=1, fail=0):
 
     return rv(name, BinomialDistribution, n, p, succ, fail)
 
+#-------------------------------------------------------------------------------
+# Beta-binomial distribution ----------------------------------------------------------
+
+class BetaBinomialDistribution(SingleFiniteDistribution):
+    _argnames = ('n', 'alpha', 'beta')
+
+    def __new__(cls, *args):
```

Comment:
Please update the new method along the lines of the refactoring done [in PR 16756](https://github.com/sympy/sympy/pull/16765).

Reply:
Done.